spec/statesman/adapters/shared_examples.rb

```diff
@@ -11,13 +11,20 @@
 #   history:          Returns the full transition history
 #   last:             Returns the latest transition history item
 #
-shared_examples_for "an adapter" do |adapter_class, transition_class|
+# rubocop:disable Metrics/LineLength
+# NOTE This line cannot reasonably be shortened.
+shared_examples_for "an adapter" do |adapter_class, transition_class, options = {}|
+  # rubocop:enable Metrics/LineLength
+
   let(:observer) do
     result = double(Statesman::Machine)
     allow(result).to receive(:execute)
```

Comment:
Could do the same `double(Statesman::Machine, execite: nil)` trick here as above if you fancy changing it :P